apps/dotcom/client/src/tla/app/TldrawApp.ts

```diff
@@ -127,30 +127,49 @@ export class TldrawApp {
 	}
 
 	messages = defineMessages({
-		publish_failed: { defaultMessage: 'Unable to publish the file.' },
-		unpublish_failed: { defaultMessage: 'Unable to unpublish the file.' },
-		republish_failed: { defaultMessage: 'Unable to publish the changes.' },
-		unknown_error: { defaultMessage: 'An unexpected error occurred.' },
+		// toast title
+		mutation_error_toast_title: { defaultMessage: 'Error' },
```

Comment:
small nit here: i know this preceded this change but i think we just stick to camelCase instead of snake_case

Reply:
oh nvm - these are sent from the backend, my bad